src/components/RestaurantDetails/components/Filters/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,15 @@
+//Libraries
+import React from 'react';
+
+//CSS
+import './Checkbox.css';
+function Checkbox() {
+  return (
+    <div className='checkbox filterItem'>
+      <input type='checkbox' />
+      <p>Veg Only</p>
```

Comment:
this should come from the Prop, since Checkbox should be used as a common component